### PR TITLE
Luxon and Refactoring ES6 syntax

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
## Awesome Books, Refactor ES6 and Luxon library
- Refactoring and convert some functions into arrow Functions
- Correct use of const let 
- Use Luxon to print the date time

#  Hi dear Reviewer
giphy (1)

### CODE REVIEW

File: modules/dateTime.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+const currentTime = document.querySelector('#currentTime');
```
Comment (Kossi-stack) [inline modules/dateTime.js:3]: - [ ] kindly display the date and time on the page
Screen Shot 2022-08-08 at 10 55 56

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @LuisDavidRodriguez,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: modules/dateTime.js (hunk 1 above)
Comment (LuisDavidRodriguez) [reply modules/dateTime.js:3]: giphy (3)

Comment (LuisDavidRodriguez) [reply modules/dateTime.js:3]: Thank you so much my friend!

**Review by DammyShittu — APPROVED** [review]
Hi @LuisDavidRodriguez,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Use of luxon for date and time

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**